app/filtering.js

```diff
@@ -130,6 +130,14 @@ function registerForBeforeRequest (session, partition) {
         onBlockedInTor(details, muonCb)
         return
       }
+    } else {
+      const host = urlParse(details.url).host
+      // US-ASCII only in `.onion', so no need for locale-dependent
+      // case-insensitive comparisons.
+      if (host.toLowerCase().endsWith('.onion')) {
```

Comment:
should check `typeof host === 'string' && host.toLowerCase().endsWith('.onion')` since host may be undefined

Reply:
UGHHH WHY